### PR TITLE
fix(dotnet): add readme content

### PR DESCRIPTION
**Pull request by JerryYangKai**
Change getting started in template.

@therealjohn  Please help review. Thanks

E2E TEST: no need.

### CODE REVIEW

**Comment by JerryYangKai** [comment]
@tecton  Do we need to add the link or info about SDK in getting started?

**Comment by tecton** [comment]
> @tecton Do we need to add the link or info about SDK in getting started?

Yes, I think we can add a link to the README of .NET SDK.

**Comment by JerryYangKai** [comment]
> > @tecton Do we need to add the link or info about SDK in getting started?
> 
> Yes, I think we can add a link to the README of .NET SDK.

AHAHA, could you please provide one or two lines so I can add it into readme.

File: templates/blazor-base/csharp/default/GettingStarted.txt [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+Before you run or Start Debugging for the first time, follow these steps:
 
-Learn more about developing Teams applications with Blazor at https://aka.ms/blazor-teams-app-dev.
+1. If the project contains Bot capability, check the Ngrok is running in your local environment. If the project only contains Tab capability, it doesn't need Ngrok.
+
+2. Select the Project > TeamsFx > Configure Microsoft Teams app. When prompted, sign in your M365 account. You can find the log in Output window.
+ 
+3. Select Debug > Start Debugging or press F5.
+
+4. After press F5, in the browser, click Add to add your app to Teams.
```
Comment (tecton) [inline templates/blazor-base/csharp/default/GettingStarted.txt:22]: You can add the sentence like:
If you want to know more about SDK usage, please check the GitHub page(https://github.com/OfficeDev/TeamsFx/tree/dev/packages/dotnet-sdk).

File: templates/blazor-base/csharp/default/GettingStarted.txt [hunk 2]
```diff
@@ -1,12 +1,25 @@
 Welcome to the Teams Toolkit! Here are some tips to get started building your app.
 
+Prerequisites
```
Comment (therealjohn) [inline templates/blazor-base/csharp/default/GettingStarted.txt:3]: To debug a Bot capability in a Teams app, you'll need to setup [ngrok](https://ngrok.com/). ngrok is used to help forward external messages from Azure Bot Framework to your locally running machine.

After installing ngrok, use a Command Prompt to run this command: ngrok http 2544

File: templates/blazor-base/csharp/default/GettingStarted.txt [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
+----------
+
+If the project contains Bot capability, you will need to setup Ngrok(https://ngrok.com/) in order to tunnel from the Teams client to localhost.
+
+After install Ngrok please run the command in Command Prompt: ngrok http 2544
+
 
 Quick start
```
Comment (therealjohn) [inline templates/blazor-base/csharp/default/GettingStarted.txt:11]: Before you run or Start Debugging for the first time, follow these steps:

1. If your project only contains a Tab capability, skip to step #3.

2. For projects that contain a Bot capability, first use a Command Prompt to run this command: ngrok http 2544.

3. Select the Project > TeamsFx > Configure Microsoft Teams app menu in Visual Studio. When prompted, sign in using your M365 account.
 
4. Select the Debug > Start Debugging menu in Visual Studio or press F5. Visual Studio will launch the app inside Microsoft Teams using a browser.

5. Once Microsoft Teams is loaded, select the Add button to install your app in Teams.

**Review by therealjohn — CHANGES_REQUESTED** [review]
I've added some suggestions inline and for the Q&A section, I have these suggestions: 

1. How do I request more permissions and scopes in a Graph call?
   If you have selected Personal Tab capability and you want to change the scope in a Graph call, you can first change the parameter _scope in Components/Graph.razor. Then go to Azure Portal -> Azure Active Directory -> App Registrations, find the AAD application Toolkit created for you (default name is the same as the project name), under Manage, select API Permissions. Select Add a permission > Microsoft Graph > Delegated permissions, then add the permissions you want.

2. Why is my bot not working in Teams?
   Please check if the ngrok is running correctly in your local environment. ngrok will expire every 2 hours and you need to restart it unless you log in with a paid subscription. After the restart, the endpoint will change and you need to click the command Project > TeamsFx > Configure Microsoft Teams app to update the new endpoint.

3. Why does the command Configure Microsoft Teams app show that ngrok is not running correctly?
   We only support the ngrok is running with default settings. Please make sure the ngrok web interface is http://127.0.0.1:4040.

**Review by hund030 — APPROVED** [review]

**Pull request merged**